lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1064,17 +1091,20 @@ func (w *WalletKit) prepareSweepParams(in *BumpFeeRequest,
 	// NOTE: if this input doesn't exist and the new budget is not
 	// specified, the params would have a zero budget.
 	params := sweep.Params{
-		Immediate:       immediate,
-		StartingFeeRate: feerate,
-		DeadlineHeight:  deadline,
-		Budget:          budget,
+		Immediate:      immediate,
+		DeadlineHeight: deadline,
+		Budget:         budget,
 	}
 
-	if ok {
-		log.Infof("[BumpFee]: bumping fee for existing input=%v, old "+
-			"params=%v, new params=%v", op, inp.Params, params)
+	// We only set the starting fee rate if it was specified else we keep
+	// the existing one.
+	if !feerate.IsNone() {
```

Comment:
This is a change to the prior version, we only overwrite the starting fee-rate of an existing sweep if a feerate was set in `conf_target` or `sat_per_vbyte`

Reply:
Can't really say if this is correct or not... Deferring to @yyforyongyu here.
But nit: can use `feeRate.IsSome()` instead.

Reply:
for example imagine we overwrite the starting fee to 0, and all of a sudden we allow inputs which where formerly disallowed see here:

https://github.com/lightningnetwork/lnd/blob/457a245a4e2cb49df8d2d34c0b189830619f9d41/sweep/aggregator.go#L246-L258